server.js

```diff
@@ -0,0 +1,6 @@
+
+import { app } from './app.js'
+
+app.listen({ port: process.env.PORT || 3000 }, function () {
+  console.log('server is running on port 3000')
```

Comment:
Don't use `console.log` at all in "production", use real logger.
In this case you don't need to log because you pass `logger: true` to  `Fastify` options

Reply:
done